### PR TITLE
Multi pane editing

### PR DESCRIPTION
**Pull request by dabbott**
Except this PR doesn't actually let you add multiple tabs. It just *supports* it. But it is big enough already.

### CODE REVIEW

File: web/src/scripts/containers/Workspace.jsx [hunk 1]
```diff
@@ -111,7 +111,42 @@ const mapDispatchToProps = (dispatch) => ({
   storyboardActions: bindActionCreators(storyboardActions, dispatch)
 })
 
-class Workspace extends Component {
+let newSceneCounter = 1
+
+class NewSceneButton extends Component {
```
Comment (dabbott) [inline web/src/scripts/containers/Workspace.jsx]: Should this still be here?

Comment (gdowens) [inline web/src/scripts/containers/Workspace.jsx]: @gabergg ?

File: web/src/scripts/actions/compositeFileActions.js [hunk 2]
```diff
@@ -31,10 +32,11 @@ export const closeTabWindow = (closeTabId) => async (dispatch, getState) => {
 
 	const tabs = getState().ui.tabs
 	const tabToFocus = tabs.CENTER && TabUtils.determineTabToFocus(tabs.CENTER.tabIds, closeTabId, tabs.CENTER.focusedTabId)
+  const fileToFocus = tabToFocus && URIUtils.withoutProtocol(tabToFocus)
```
Comment (gdowens) [inline web/src/scripts/actions/compositeFileActions.js]: Is tabbing the only place where we have the scheme attached to resource?

File: web/src/scripts/components/editor/Editor.jsx [hunk 3]
```diff
@@ -25,28 +25,67 @@ import CodeMirrorComponent from './CodeMirrorComponent'
...
-    this.attachMiddleware(props.middleware, props.decoDoc)
+
+    this.state = {
+      linkedDoc: this.createLinkedDoc(props.decoDoc)
+    }
+
+    this.attachMiddleware(props.middleware, props.decoDoc, this.state.linkedDoc)
+  }
+
+  createLinkedDoc(decoDoc) {
```
Comment (gdowens) [inline web/src/scripts/components/editor/Editor.jsx]: Hmm.. does it make sense to have this logic in Editor? Where would we put it if not?

File: web/src/scripts/containers/ProjectNavigator.jsx [hunk 4]
```diff
@@ -79,7 +79,7 @@ class ProjectNavigator extends Component {
   onDoubleSelectFile = (e, node) => {
     const {path: filepath, type} = node
 
-    this.props.tabActions.makeTabPermanent(CONTENT_PANES.CENTER, filepath)
+    this.props.tabActions.makeTabPermanent(CONTENT_PANES.CENTER, 'file://' + filepath)
```
Comment (gdowens) [inline web/src/scripts/containers/ProjectNavigator.jsx]: (>_>) it's getting weird man

File: web/src/scripts/containers/Workspace.jsx (hunk 1 above)
Comment (gabergg) [inline web/src/scripts/containers/Workspace.jsx]: killit

**Pull request merged**